docs/intelligentapps/overview.md

```diff
@@ -72,7 +71,42 @@ You can also install the AI Toolkit extension manually from the Visual Studio Co
 > Check the **What's New** page after installation to see detailed features for each version.
 * After successful installation, the AI Toolkit icon appears in the Activity Bar.
 
-   ![Screenshot showing the AI Toolkit icon in the VS Code Activity Bar after installation](./images/overview/install-4.png)
+
+
+## Explore Azure AI Toolkit
+
+The Azure AI Toolkit opens in its own view, with the Azure AI Toolkit Icon now displayed on the VS Code Navbar. The extension has several main sections: My Resources, Model Tools, Agent and Workflow Tools, MCP Workflow, and Help and Feedback.
+
+   ![Screenshot showing the AI Toolkit Extension with highlighted sections.":::
+](./images/overview/initial-view.png)
+
+- **My Resources**: This section contains the resources you have access to in AI Toolkit. The **My Resources** section is the main view for interacting with your Azure AI resources. It contains the following subsections:
+  - **Models**: This section contains the models you can use to build and deploy for your AI applications. The **Models** view is where you can find your deployed models in AI Toolkit.
+  - **Agents**: This section contains your AI Toolkit deployed agents.
+  - **MCP Servers**: This section contains the MCP Servers you're working with in AI Toolkit.
+
+- **Model Tools**: This section contains the model tools you can use to build and deploy your AI applications. The **Model Tools** view is where you can find the tools available to deploy and then work with your deployed models. It contains the following subsections:
+    - **Model Catalog**: The link to the model catalog lets you discover and access AI models from multiple sources including GitHub, ONNX, Ollama, OpenAI, Anthropic, and Google. Compare models side-by-side and find the right model for your use case.
```

Comment:
"The link to" sounds weird. Can we just use "The model catalog lets you discover..." ?
Similar for the next list items.